jenkins/PerfCI_Chaos/03_PerfCI_Choas_VMs_Deployment/Jenkinsfile

```diff
@@ -0,0 +1,206 @@
+pipeline {
```

Comment:
I just noticed that "chaos" is misspelled as "choas" in the directory name.  Please fix.

Reply:
Fixed